### PR TITLE
Fix image digest to handle tags

### PR DESCRIPTION
**Pull request by david4096**
Currently only works with branches. To work with tags, you must go look up the commit a tag references.

Also adds a slightly more useful error message when the digest isn't found.

### CODE REVIEW

File: scripts/webservice-image-digest.py [hunk 1]
```diff
@@ -20,14 +20,19 @@
 
 DOCKER_TAG_BASE = "quay.io/dockstore/dockstore-webservice"
 
+def get_commit_from_tag_url(tag_url):
+  # takes a tag url and gets the commit hash it is pointed at
```
Comment (coverbeck) [inline scripts/webservice-image-digest.py:24]: Will this work for non-annotated tags? We typically won't have them, as Hubflow always creates an annotated tag, but there might be a test case or something where we try with a lightweight tag.

**Review by coverbeck — COMMENTED** [review]
Not crucial, but might as well make sure it works with lightweight tags.

File: scripts/webservice-image-digest.py (hunk 1 above)
Comment (david4096) [reply scripts/webservice-image-digest.py:24]: Added another commit to handle simple tags

**Review by coverbeck — COMMENTED** [review]
Looks good, but needs to target a hotfix branch (that hasn't been created yet).

And a test as Denis mentioned, but that can be a followup.

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**